src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,47 @@
 package mate.jdbc;
 
+import java.util.ArrayList;
+import java.util.List;
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Car;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.service.CarService;
+import mate.jdbc.service.DriverService;
+import mate.jdbc.service.ManufacturerService;
+
 public class Main {
+    private static final Injector INJECTOR = Injector.getInstance("mate.jdbc");
+
     public static void main(String[] args) {
-        // test your code here
+        CarService carService = (CarService) INJECTOR.getInstance(CarService.class);
+        DriverService driverService = (DriverService) INJECTOR.getInstance(DriverService.class);
+        ManufacturerService manufacturerService
+                = (ManufacturerService) INJECTOR.getInstance((ManufacturerService.class));
+        Driver petro = driverService.get(1L);
+        Driver ivan = driverService.get(2L);
```

Comment:
Let's try to not hardcode id values in the code, because it can be changed.
Call firstly `Driver driver = driverService.create();`
and then get id from the driver:
```suggestion
        Driver ivan = driverService.get(driver.getId());
```
Please fix all similar places.